corehq/apps/userreports/management/commands/rebuild_tables_by_domain.py

```diff
@@ -6,6 +6,20 @@
     StaticDataSourceConfiguration,
 )
 
+from django.db import connections
+from django.db.utils import ProgrammingError
+
+from corehq.sql_db.connections import get_icds_ucr_citus_db_alias
+
+
+def _run_custom_sql_script(command, day=None):
+    db_alias = get_icds_ucr_citus_db_alias()
+    if not db_alias:
+        return
+
+    with connections[db_alias].cursor() as cursor:
+        cursor.execute(command, [day])
```

Comment:
shouldn't this be an empty list if `day is None`?

Reply:
yes, correct.